dev/reference/configuration/tikv-server/configuration-file.md

```diff
@@ -89,6 +89,43 @@ TiKV 配置文件比命令行参数支持更多的选项。你可以在 [etc/con
 + 单位：KB|MB|GB
 + 最小值：1KB
 
+## readpool
+
+读取线程池相关的配置项。
+
+### `unify-thread-pool`
+
++ 是否使用单个线程池处理所有的读请求。
++ 默认值：true
+
+## readpool.unified
+
+统一处理读请求的线程池相关的配置项。
+以下配置只在 `unify-read-pool` 是 `true` 时生效。
+
+### `min-thread-count`
+
++ 统一处理读请求的线程池最少的线程数量。
++ 默认值：1
```

Comment:
Do we have a max value?

Reply:
No. It's valid as long as it is less than `max-thread-count` (which is trivial)

Reply:
PTAL @AndreMouche